scripts/update_package_version.sh

```diff
@@ -10,6 +10,12 @@
 # Prompt changed packages to manually set the new version
 
 if [[ "$GITHUB_RUN_ID" ]]; then
+  # do not attempt versioning if the NODE_AUTH_TOKEN is missing
+  if [[ -z "$NODE_AUTH_TOKEN" ]]; then
```

Comment:
Just a note: this is under assumption that no one has NODE_AUTH_TOKEN except the people actually wants to publish it.

Reply:
Seems like there is some sort of placeholder for `NODE_AUTH_TOKEN` under forked repos as well. So, this check does NOT work. We need to come up with a better solution.

Reply:
Now using a job level conditional within the workflow yaml file